### ISSUE TITLE
[REQUEST] Can't Modify RetroArch Save Settings

### ISSUE THREAD

**Issue by ndplymale** [jimgraygit/Onion#245]
Hello OnionOS team, 

I wanted to report that it appears the RetroArch page for saving cannot be modified with the latest version. (RetroArch App > Settings > Saving) 

It seems likely that this decision was made so that the launcher can work properly. However, if the launcher is disabled, can we please get access back to these settings? I'm not too fond of quickly entering/exiting a game and it creating a save state every time.

**Comment by Sichroteph** [comment]
Nothing changed across the versions. As always when changing a global setting, you need to close the content first, or to open RetroArch from apps. You will be able to save your configuration then.

**Comment by ndplymale** [comment]
Thanks for replying @Sichroteph! 

After checking into RetroArch a bit further, I found that I was forgetting to save the current configuration before exiting. I'm still new to RA and that wasn't immediately clear and thought the settings were locked. My apologies! Also, in the process of tinkering, I messed up the OS and couldn't launch games anymore. Oops. Decided to delete the system files and run the  .tmp_update process. 

It worked immaculately. 

When I started it back up, I noticed I was a few versions ahead since the time I downloaded it about a week ago! The game browsing lists look SO much better with the image updates. (gradient + no cutoff!) Nice work Onion devs!!
